apps/condo/domains/marketplace/components/Invoice/InvoiceRowsTable.tsx

```diff
@@ -113,7 +114,7 @@ export const InvoiceRowsTable: React.FC<InvoiceRowsTableProps> = ({ invoice }) =
     const ContractPriceMessage = intl.formatMessage({ id: 'pages.condo.marketplace.invoice.form.contractPrice' }).toLowerCase()
 
     const currencyCode = get(invoice, 'currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
-    const rows = useMemo(() => get(invoice, 'rows'), [invoice])
+    const rows = useMemo(() => get(invoice, 'rows') || [], [invoice])
```

Comment:
Or another solution if you're using get
```suggestion
    const rows = useMemo(() => get(invoice, 'rows', []), [invoice])
```